com.unity.perception/Documentation~/FAQ/FAQ.md

```diff
@@ -0,0 +1,584 @@
+
+<details>
+  <summary><strong>Q: How can I remove or add new labels to objects at runtime?</strong></summary><br>
+
+This can be achieved through modifying the `labels` list of the `Labeling` component. The key is to call `RefreshLabeling()` on the component after making any changes to the labels. Example:
+
+```C#
+var labeling = gameObject.GetComponent<Labeling>();
+labeling.labels.Clear();
+labeling.labels.Add("new-label");
+labeling.RefreshLabeling();
+```
+Keep in mind that any new label added with this method should already be present in the Label Config attached to the Labeler that is supposed to label this object.
+
+---
+</details>
+
+<details>
+  <summary><strong>Q: Is it possible to label only parts of an object or assign different labels to different parts of objects?</strong></summary><br>
+
+  Labeling works on the GameObject level, so to achieve the scenarios described here, you will need to break down your main object into multiple GameObjects parented to the same root object, and add `Labeling` components to each of the inner objects, as shown below.
+
+<p align="center">
+<img src="images/inner_objects.png" width="800"/>
+</p>
+  
+  Alternatively, in cases where parts of the surface of the object need to be labeled (e.g. decals on objects), you can add labeled invisible surfaces on top of these sections. These invisible surfaces need to have a fully transparent material. To create an invisible material:
+
+  * Create a new material (***Assets -> Create -> Material***) and name it `TransparentMaterial`
+  * Set the **Surface Type** for the material to **Transparent**, and set the alpha channel of the **Base Map** color to 0.
+    * For HDRP: In addition to the above, disable **Preserve specular lighting** 
+   
+  An example labeled output for an object with separate labels on inner objects and decals is shown below:
+
+<p align="center">
+<img src="images/inner_labels.gif" width="600"/>
+</p> 
+
+---
+</details>
+
+<details>
+  <summary><strong>Q: When visible surfaces of two objects are fully aligned, the bounding boxes seem to blink in and out of existence from one frame to another. Why is that?</strong></summary><br>
+
+This is due to a common graphics problem called *z-fighting*. This occurs when the shader can't decide which of the two surfaces to draw on top of the other, since they both have the exact same distance from the camera. To fix this, simply move one of the objects slightly so that the two problematic surfaces do not fully align.
+
+---
+</details>
+
+
+## <a name="randomization">Randomization</a>
+
+<details>
+  <summary><strong>Q: How can I have multiple sets of prefabs in a foreground placement Randomizer, and on every Iteration select one from each set?</strong>
+  </summary><br>
+
+  This question is an example of more complex functionality that can be achieved by applying slight modifications to the provided sample Randomizers, or by creating completely custom ones by extending the `Randomizer` class. 
```

Comment:
This definitely seems like a randomizer that we should add to @aryan-mann's randomizer list

Reply:
yes, we will need to put some of these in that library.